flow/layers/layer_tree.h

```diff
@@ -50,7 +50,12 @@ class LayerTree {
   const SkISize& frame_size() const { return frame_size_; }
   float device_pixel_ratio() const { return device_pixel_ratio_; }
 
-  void RecordBuildTime(fml::TimePoint build_start, fml::TimePoint target_time);
+  void RecordBuildTime(fml::TimePoint vsync_start,
+                       fml::TimePoint build_start,
+                       fml::TimePoint target_time);
+  fml::TimePoint vsync_start() const { return vsync_start_; }
+  fml::TimeDelta vsync_overhead() const { return build_start_ - vsync_start_; }
+  fml::TimeDelta vsync_time() const { return build_finish_ - vsync_start_; }
```

Comment:
`vsync_time` this isn't accurate. The name suggests that this is the time between adjacent vsync signals, which this isn't. Why do we need this method?

Reply:
Removed it.

Reply:
I shouldn't have removed it..

Reply:
So remove it again.